netbench/cdk/src/main/java/com/aws/ClientServerStack.java

```diff
@@ -91,14 +92,33 @@ public ClientServerStack(final Construct parent, final String id, final ClientSe
         
         cluster.addAsgCapacityProvider(asgProvider);
 
+        
+        /*
```

Comment:
I think generally it would be better to delete all the commented out code until it is ready to be uncommented out, since we'd want to make sure it is reviewed properly at the time it is actually going to be applied

Reply:
That makes sense, I'll remove it